src/compiler/checker.ts

```diff
@@ -1526,17 +1534,53 @@ module ts {
                     }
                 }
 
+                function writeSymbolTypeReference(symbol: Symbol, typeArguments: Type[], pos: number, end: number) {

```

Comment:
Can you rename pos and end, or add a comment saying what the pos and end are referring to?